modules/oe_content_project/config/install/field.field.node.oe_project.oe_project_dates.yml

```diff
@@ -0,0 +1,20 @@
+langcode: en
+status: true
+dependencies:
+  config:
+    - field.storage.node.oe_project_dates
+    - node.type.oe_project
+  module:
+    - datetime_range
+id: node.oe_project.oe_project_dates
+field_name: oe_project_dates
+entity_type: node
+bundle: oe_project
+label: 'Project period'
+description: 'If project has no end date leave it empty or use the same start date value'
```

Comment:
![image](https://user-images.githubusercontent.com/22004498/82888389-86219c00-9f49-11ea-87c7-eeab63721ddb.png)
I tried to leave the end date empty, I can't save the form due to validation error: "This value should not be null."

Reply:
After discussing internally we have decided to change the description to adapt to Drupal's behavior. We need to check if it makes sense to not have an and date for projects at all.